patches/chromium/.patches

```diff
@@ -139,3 +139,5 @@ build_disable_thin_lto_mac.patch
 build_add_public_config_simdutf_config.patch
 revert_code_health_clean_up_stale_macwebcontentsocclusion.patch
 build_remove_vr_directx_helpers_dependency.patch
+check_for_unit_to_activate_before_notifying_about_success.patch
```

Comment:
Seems like a bad rebase, `check_for_unit_to_activate_before_notifying_about_success.patch` line should be removed.

Reply:
Thanks, fixed!